### ISSUE TITLE
Compilation error when passing a variable with name 'true*' or 'false*'

### ISSUE THREAD

**Issue by folex** [fluencelabs/aqua#190]
`test.aqua`:
```scala
service My("my"):
    test(value: ?bool)
    identity(value: bool) -> bool

func test():
    true_value: *bool
    true_value <- My.identity(true)
    My.test(true_value)
```

Compiling this code gives an error:
```
> aqua-cli -i . -o ./typescript

java -jar .../aqua-cli.jar -m node_modules -i . -o ./typescript
[info] Aqua Compiler 0.1.8-160

.../test.aqua:8:5
5  func test():
6      true_value: *bool
7      true_value <- My.identity(true)
8      My.test(true_value)
       ^==================
       Syntax error, expected: OneOfStr(151,List(<-, catch, co, else, for, if, on, otherwise, par, try)), InRange(151,a,z), InRange(153,	,	), InRange(153, , ), InRange(163,),))
9
10
11
```

However, this one compiles just fine:
```scala
service My("my"):
    test(value: ?bool)
    identity(value: bool) -> bool

func test():
    bool_value: *bool
    bool_value <- My.identity(true)
    My.test(bool_value)
```

**Comment by alari** [comment]
This happens on the parser level for variables: I suppose literal parsing has no backtrack option.

**Comment by alari** [comment]
Fixed with #198
